test/E2ETest/Microsoft.Test.E2E.AspNet.OData/QueryComposition/OrderByTests.cs

```diff
@@ -103,6 +104,24 @@ private static IEdmModel GetModel(WebRouteConfiguration configuration)
                             previousElement.WorkAddress.CountryOrRegion.Name.CompareTo(currentElement.WorkAddress.CountryOrRegion.Name) < 1);
             }
         }
+
+        [Fact]
+        public async Task CanOrderByInOperator()
+        {
```

Comment:
I am planning to revert this as $orderby does not appear to support expressions (only properties so not even boolean). Just wanted to show this code to make a comment here.

Reply:
Exception message: "Only ordering by properties is supported for non-primitive collections. Expressions are not supported."